src/pointer.cr

```diff
@@ -129,6 +129,16 @@ struct Pointer(T)
     (self + offset).value = value
   end
 
+  {% if flag?(:x86_64) %}
+    private def to_size_t(size : Int) : UInt64
+        size.to_u64
+    end
+  {% else %}
+    private def to_size_t(size : Int) : UInt32
+        size.to_u32
+    end
+  {% end %}
+
```

Comment:
No. `Intrinsics` method signatures doesn't related to `LibC::SizeT`.

This PR is short one, without changing of rules of the game.